src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UserManagement/UsersController.cs

```diff
@@ -107,17 +82,11 @@ public async Task<ActionResult<bool>> DeleteUserReleaseRole(Guid userReleaseRole
         /// <returns>Id and Title of the release</returns>
         [HttpGet("user-management/releases")]
         [ProducesResponseType(200)]
-        [ProducesResponseType(404)]
-        public async Task<ActionResult<IdTitlePair>> GetReleases()
+        public async Task<ActionResult<List<IdTitlePair>>> GetReleases()
```

Comment:
I think we could open up a new tech debt ticket to remove this as it feels like it shouldn't be the responsibility of this controller and is probably duplicating stuff elsewhere. Also if it's fetching Releases rather than Publications that's going to be a very long list to display eventually.